tensorflow/contrib/BUILD

```diff
@@ -126,6 +126,7 @@ py_library(
     }) + select({
         "//tensorflow:with_ignite_support": [
             "//tensorflow/contrib/ignite",
+            "//tensorflow/contrib/igfs",
```

Comment:
Can we combine these two modules into one? IGFS seems like it would fit under `"//tensorflow/contrib/ignite"`.

Reply:
I would prefer to have them in separate modules by several reasons. First of all, they have very different functionality (`"//tensorflow/contrib/ignite"` provides an ability to read data from Apache Ignite Cache while `"//tensorflow/contrib/igfs"` provides allows to use file system, based on Apache Ignite). User may want to use only one feature, but not other. It looks like very reasonable in some cases. I think it's a case when "write programs that do one thing and do it well" is good, isn't it? And by the way, after `contrib` sunsetting it makes even more sense to have them in separate packages.

Are you agree?

Reply:
Sounds reasonable. I merged two modules together.